The job **`i686-gnu`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/9219099624/job/25363704590) [(plain)](https://github.com/rust-lang-ci/rust/commit/ef87c427587ca3852b0bc7e533969cc02e7cfd7d/checks/25363704590/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
---- [run-make] tests/run-make/libtest-padding stdout ----

error: make failed
status: exit status: 2
command: cd "/checkout/tests/run-make/libtest-padding" && env -u CARGO_MAKEFLAGS -u MAKEFLAGS -u MFLAGS -u RUSTFLAGS AR="ar" CC="cc -ffunction-sections -fdata-sections -fPIC -m32 -march=i686" CXX="c++ -ffunction-sections -fdata-sections -fPIC -m32 -march=i686" HOST_RPATH_DIR="/checkout/obj/build/i686-unknown-linux-gnu/stage2/lib" LD_LIB_PATH_ENVVAR="LD_LIBRARY_PATH" LLVM_BIN_DIR="/checkout/obj/build/i686-unknown-linux-gnu/llvm/bin" LLVM_COMPONENTS="aarch64 aarch64asmparser aarch64codegen aarch64desc aarch64disassembler aarch64info aarch64utils aggressiveinstcombine all all-targets analysis arm armasmparser armcodegen armdesc armdisassembler arminfo armutils asmparser asmprinter avr avrasmparser avrcodegen avrdesc avrdisassembler avrinfo binaryformat bitreader bitstreamreader bitwriter bpf bpfasmparser bpfcodegen bpfdesc bpfdisassembler bpfinfo cfguard codegen codegentypes core coroutines coverage csky cskyasmparser cskycodegen cskydesc cskydisassembler cskyinfo debuginfobtf debuginfocodeview debuginfodwarf debuginfogsym debuginfologicalview debuginfomsf debuginfopdb demangle dlltooldriver dwarflinker dwarflinkerclassic dwarflinkerparallel dwp engine executionengine extensions filecheck frontenddriver frontendhlsl frontendoffloading frontendopenacc frontendopenmp fuzzercli fuzzmutate globalisel hexagon hexagonasmparser hexagoncodegen hexagondesc hexagondisassembler hexagoninfo hipstdpar instcombine instrumentation interfacestub interpreter ipo irprinter irreader jitlink libdriver lineeditor linker loongarch loongarchasmparser loongarchcodegen loongarchdesc loongarchdisassembler loongarchinfo lto m68k m68kasmparser m68kcodegen m68kdesc m68kdisassembler m68kinfo mc mca mcdisassembler mcjit mcparser mips mipsasmparser mipscodegen mipsdesc mipsdisassembler mipsinfo mirparser msp430 msp430asmparser msp430codegen msp430desc msp430disassembler msp430info native nativecodegen nvptx nvptxcodegen nvptxdesc nvptxinfo objcarcopts objcopy object objectyaml option orcdebugging orcjit orcshared orctargetprocess passes powerpc powerpcasmparser powerpccodegen powerpcdesc powerpcdisassembler powerpcinfo profiledata remarks riscv riscvasmparser riscvcodegen riscvdesc riscvdisassembler riscvinfo riscvtargetmca runtimedyld scalaropts selectiondag sparc sparcasmparser sparccodegen sparcdesc sparcdisassembler sparcinfo support symbolize systemz systemzasmparser systemzcodegen systemzdesc systemzdisassembler systemzinfo tablegen target targetparser textapi textapibinaryreader transformutils vectorize webassembly webassemblyasmparser webassemblycodegen webassemblydesc webassemblydisassembler webassemblyinfo webassemblyutils windowsdriver windowsmanifest x86 x86asmparser x86codegen x86desc x86disassembler x86info x86targetmca xray" LLVM_FILECHECK="/checkout/obj/build/i686-unknown-linux-gnu/llvm/build/bin/FileCheck" PYTHON="/usr/bin/python3" RUSTC="/checkout/obj/build/i686-unknown-linux-gnu/stage2/bin/rustc" RUSTDOC="/checkout/obj/build/i686-unknown-linux-gnu/stage2/bin/rustdoc" RUST_BUILD_STAGE="stage2-i686-unknown-linux-gnu" S="/checkout" TARGET="i686-unknown-linux-gnu" TARGET_RPATH_DIR="/checkout/obj/build/i686-unknown-linux-gnu/stage2/lib/rustlib/i686-unknown-linux-gnu/lib" TMPDIR="/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding" "make"
--- stdout -------------------------------
LD_LIBRARY_PATH="/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding:/checkout/obj/build/i686-unknown-linux-gnu/stage2/lib:/checkout/obj/build/i686-unknown-linux-gnu/stage0-bootstrap-tools/i686-unknown-linux-gnu/release/deps:/checkout/obj/build/i686-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/i686-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding -L /checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding  -Ainternal_features --test tests.rs
LD_LIBRARY_PATH="/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding:/checkout/obj/build/i686-unknown-linux-gnu/stage2/lib/rustlib/i686-unknown-linux-gnu/lib:/checkout/obj/build/i686-unknown-linux-gnu/stage0-bootstrap-tools/i686-unknown-linux-gnu/release/deps:/checkout/obj/build/i686-unknown-linux-gnu/stage0/lib" /checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding/tests --test-threads=1 | sed 's%[0-9,\.]\{1,\} ns/iter (+/- [0-9,\.]\{1,\})%?? ns/iter (+/- ??)%' | sed 's%finished in [0-9\.]\{1,\}%finished in ??%' > "/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding"/test.stdout
diff -u --strip-trailing-cr "/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding"/test.stdout test.stdout
LD_LIBRARY_PATH="/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding:/checkout/obj/build/i686-unknown-linux-gnu/stage2/lib/rustlib/i686-unknown-linux-gnu/lib:/checkout/obj/build/i686-unknown-linux-gnu/stage0-bootstrap-tools/i686-unknown-linux-gnu/release/deps:/checkout/obj/build/i686-unknown-linux-gnu/stage0/lib" /checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding/tests --test-threads=1 --bench | sed 's%[0-9,\.]\{1,\} ns/iter (+/- [0-9,\.]\{1,\})%?? ns/iter (+/- ??)%' | sed 's%finished in [0-9\.]\{1,\}%finished in ??%' > "/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding"/bench.stdout
diff -u --strip-trailing-cr "/checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding"/bench.stdout bench.stdout
--- /checkout/obj/build/i686-unknown-linux-gnu/test/run-make/libtest-padding/libtest-padding/bench.stdout 2024-05-24 06:43:03.975409342 +0000
+++ bench.stdout 2024-05-24 05:26:43.658786273 +0000
@@ -2,8 +2,8 @@
 test short_test_name ... ignored
 test this_is_a_really_long_test_name ... ignored
 test this_is_a_really_long_test_name ... ignored
-test short_bench_name                 ... bench:          ?? ns/iter (+/- ??)
-test this_is_a_really_long_bench_name ... bench:          ?? ns/iter (+/- ??)
+test short_bench_name                 ... bench:           ?? ns/iter (+/- ??)
+test this_is_a_really_long_bench_name ... bench:           ?? ns/iter (+/- ??)
 test result: ok. 0 passed; 0 failed; 2 ignored; 2 measured; 0 filtered out; finished in ??s
------------------------------------------
--- stderr -------------------------------
make: *** [Makefile:12: all] Error 1
```

</details>

--- a/tests/run-make/libtest-padding/tests.rs
+++ b/tests/run-make/libtest-padding/tests.rs
@@ -8,11 +8,7 @@ fn short_test_name() {}
 fn this_is_a_really_long_test_name() {}
 
 #[bench]
-fn short_bench_name(b: &mut test::Bencher) {
-    b.iter(|| 1);
-}
+fn short_bench_name(b: &mut test::Bencher) {}
 
 #[bench]
-fn this_is_a_really_long_bench_name(b: &mut test::Bencher) {
-    b.iter(|| 1);
-}
+fn this_is_a_really_long_bench_name(b: &mut test::Bencher) {}

